### DISCUSSION TITLE
is there a way to tweet with line break?

### DISCUSSION THREAD

**Discussion by rcnpat** [tweepy/tweepy#1638] [Questions] [answered]
how can i tweet like this

`hello world`
`from tweepy`

instead of this

`hello world from tweepy`

thanks in advance

**Comment by Harmon758** [comment, marked as answer]
Yes, line breaks are represented by their escape sequence, `\n`, in strings, so using [`API.update_status`](https://tweepy.readthedocs.io/en/v3.10.0/api.html#API.update_status) with `"hello world\nfrom tweepy"` would work.
